### PR TITLE
[Mobile app] Adding Section for Contributors.

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Updates the config file and .env file to remove unused environment variables.
- [x] Update README file.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Test out the app to ensure it works as expected.
- Review the changed files.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [MOB-153](https://airqoteam.atlassian.net/browse/MOB-153) 
-

### CODE REVIEW

File: mobile/README.md [hunk 1]
```diff
@@ -35,6 +35,25 @@
...
+  - It's important to first read our guidelines [here](/../CONTRIBUTING.md).
+  - Thereafter, proceed to create your own necessary configuration files. These include;
+    - Add an `.env file`, which contain credentials in key-value format for services used by the app. For development, save the file as `.env.dev` while for production, save it as `.env.prod`. 
+      - Here's what your .env file would look like;
+        ```bash
+        AIRQO_API_URL= https://platform.airqo.net/api/v1/
+        AIRQO_API_URL_V2= https://platform.airqo.net/api/v2/
+        AIRQO_API_TOKEN = Get this token by following the respective guide below.
+        GOOGLE_ANDROID_API_KEY_= Get this by following the respective guide below.
+        GOOGLE_IOS_API_KEY_v1 = Get this key by following the respective guide below.
```
Comment (NoahNsimbe) [inline mobile/README.md:48]: I don't think these two should be part of the .env @BenjaminSsempala

File: mobile/README.md [hunk 2]
```diff
@@ -35,6 +35,25 @@
...
+        SEARCH_API_KEY=
+        ```
+      - Get the following keys by following documentaion from the respectice sources;
+        - [AIRQO_API_TOKEN](https://staging-docs.airqo.net/#/../api/users?id=login),
+        - [GOOGLE_ANDROID_API_KEY](https://support.google.com/googleapi/answer/6158862),
+        - [GOOGLE_IOS_API_KEY_v1](https://developers.google.com/maps/documentation/ios-sdk/get-api-key)
```
Comment (NoahNsimbe) [inline mobile/README.md:54]: Same where

**Review by NoahNsimbe — COMMENTED** [review]
I think the contents of the .env are :
```bash
AIRQO_API_TOKEN
SEARCH_API_KEY
AIRQO_API_URL
AIRQO_API_USER_EXISTS_URL
FAV_PLACES_COLLECTION
KYA_COLLECTION
USERS_NOTIFICATION_COLLECTION
USERS_ANALYTICS_COLLECTION
USERS_COLLECTION
USERS_KYA_COLLECTION
USERS_PROFILE_PICTURE_COLLECTION
PREF_ON_BOARDING_PAGE
SENTRY_DSN
```

For example

```dart
  static String get airqoApiToken => dotenv.env['AIRQO_API_TOKEN'] ?? '';
  static String get searchApiKey => dotenv.env['SEARCH_API_KEY'] ?? '';

  static String get airqoApiUrl => dotenv.env['AIRQO_API_URL'] ??'https://api.airqo.net/api/v1/';

  static String get airqoApiUserExistsUrl => dotenv.env['AIRQO_API_USER_EXISTS_URL'] ?? '';

  static String get favPlacesCollection => dotenv.env['FAV_PLACES_COLLECTION'] ?? '';

  static String get kyaCollection => dotenv.env['KYA_COLLECTION'] ?? '';

  static String get usersNotificationCollection => dotenv.env['USERS_NOTIFICATION_COLLECTION'] ?? '';

  static String get usersAnalyticsCollection => dotenv.env['USERS_ANALYTICS_COLLECTION'] ?? '';

  static String get usersCollection => dotenv.env['USERS_COLLECTION'] ?? '';

  static String get usersKyaCollection => dotenv.env['USERS_KYA_COLLECTION'] ?? '';

  static String get usersProfilePictureStorage => dotenv.env['USERS_PROFILE_PICTURE_COLLECTION'] ?? '';

  static String get prefOnBoardingPage => dotenv.env['PREF_ON_BOARDING_PAGE'] ?? '';

  static String get sentryDsn => dotenv.env['SENTRY_DSN'] ?? '';
```

File: mobile/README.md [hunk 3]
```diff
@@ -35,6 +35,25 @@
 - Change `storeFile` variable in `key.properties` file to point to where you put `airqo-dev-keystore.jks`.
 For example if `airqo-dev-keystore.jks` is located under `/Users/example/` then `storeFile=/Users/example/airqo-dev-keystore.jks`
 
+- **For Contributors:**
```
Comment (NoahNsimbe) [inline mobile/README.md:38]: Make this a heading @BenjaminSsempala 
i.e `### **For Contributors:**`

File: mobile/README.md (hunk 1 above)
Comment (BenjaminSsempala) [reply mobile/README.md:48]: Thank you. I've removed them @NoahNsimbe

**Review by NoahNsimbe — COMMENTED** [review]
Thanks @BenjaminSsempala.  LGTM.

**Review by Mnoble-19 — APPROVED** [review]
Tested map, search and insights and all is working fine.

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**